cmd/influxd/server_integration_test.go

```diff
@@ -433,9 +476,9 @@ func runTestsData(t *testing.T, testName string, nodes Cluster, database, retent
 			expected: `{"results":[{"series":[{"name":"logs","columns":["time","event"]}]}]}`,
 		},
 		{
-			name:     "missing measurement with `GROUP BY *`",
-			query:    `select load from "%DB%"."%RP%".missing group by *`,
-			expected: `{"results":[{"error":"measurement not found: \"%DB%\".\"%RP%\".\"missing\""}]}`,
+			name:          "missing measurement with `GROUP BY *`",
+			query:         `select load from "%DB%"."%RP%".missing group by *`,
+			expectPattern: `.*measurement not found: .*missing.*`,
```

Comment:
I have seen it in practice in open source but can't site a project off the top of my head.  The user friendly part of the error message is at the beginning of the line where the user would look first.  The extra info is at the end.  I think the extra info benefits the user and us by making it easier to debug issues.  Prior to this change, we just report "measurement not found", which doesn't help the user or us much.  We don't know what measurement wasn't found and we don't know what part of the code couldn't look up the measurement.